src/patternfly/components/LogViewer/log-viewer.scss

```diff
@@ -21,11 +24,14 @@
   --pf-c-log-viewer--m-line-numbers__main--before--BackgroundColor: var(--pf-global--BorderColor--100);
 
   // List
-  // --pf-c-log-viewer__list--PaddingTop: var(--pf-global--spacer--sm);
-  // --pf-c-log-viewer__list--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-log-viewer__list--Width: 100%;
+  --pf-c-log-viewer__list--Height: auto;
 
-  --pf-c-log-viewer__scroll-container--PaddingTop: var(--pf-global--spacer--md);
-  --pf-c-log-viewer__scroll-container--PaddingBottom: var(--pf-global--spacer--md);
+  // List item
+  --pf-c-log-viewer__list-item--Right: 0;
+  --pf-c-log-viewer__list-item--Left: 0;
+  --pf-c-log-viewer__list-item--Width: 100%;
+  --pf-c-log-viewer--m-nowrap__list-item--Left: var(--pf-c-log-viewer__index--Width);
```

Comment:
If I understand correctly, this should be applied to the list but not the list-item, right? Also, this one should be the same no matter when the line is wrapped or not. It could be different when there is no line numbers, so my suggestion is that this could be set to `0` in `.pf-c-log-viewer__list` and changed to `var(--pf-c-log-viewer__index--Width)` in `.pf-m-line-numbers` modifier.